model_analyzer/config/generate/coordinate_data.py

```diff
@@ -26,6 +26,7 @@ class CoordinateData:
     def __init__(self):
         self._measurements = {}
         self._visit_counts = {}
+        self._is_measured = {}
```

Comment:
Don't you need to define what these are for type checking to work correctly?

Reply:
type checking passed 🤷‍♂️